src/commonTest/kotlin/suite/TestSuiteMultiCluster.kt

```diff
@@ -29,8 +29,8 @@ internal class TestSuiteMultiCluster {
         return "$prefix-$username-$id".toUserID()
     }
 
-    private suspend fun CoroutineScope.waitForUserID(userID: UserID) {
-        while (isActive) {
+    private suspend fun waitForUserID(userID: UserID) {
```

Comment:
I didn't know about this convention, do you have a reference I could read somewhere ? :)

Reply:
https://medium.com/@elizarov/coroutine-context-and-scope-c8b255d59055

Reply:
Thanks !